src/Magnum/DebugTools/BufferData.h

```diff
@@ -47,35 +47,45 @@ namespace Implementation {
 @brief Buffer subdata
 
 Emulates @ref GL::Buffer::subData() call on platforms that don't support it
-(such as OpenGL ES) by using @ref GL::Buffer::map().
+(such as OpenGL ES) by using @ref GL::Buffer::mapRead(). On desktop GL and
+WebGL 2.0 it's just an alias to @ref GL::Buffer::subData().
 
 @note This function is available only if Magnum is compiled with
     @ref MAGNUM_TARGET_GL "TARGET_GL" enabled (done by default). See
     @ref building-features for more information.
 
 @requires_gles30 Extension @gl_extension{EXT,map_buffer_range} in OpenGL ES
     2.0.
-@requires_gles Buffer mapping is not available in WebGL.
+@requires_webgl20 Buffer data queries or buffer mapping are not available in
+    WebGL 1.0.
 */
-template<class T> Containers::Array<T> inline bufferSubData(GL::Buffer& buffer, GLintptr offset, GLsizeiptr size) {
+template<class T = char> Containers::Array<T> inline bufferSubData(GL::Buffer& buffer, GLintptr offset, GLsizeiptr size) {
```

Comment:
Why the sudden `= char`? I want to de-template this eventually anyway, so the less API behavior changes in the meantime, the better I'd say.

Reply:
The intention was to unify `bufferData()` and `bufferSubData()`. It felt odd having to explicitly add `<char>` depending on whether I wanted part of or the entire buffer data. But I can drop that commit, just wasn't aware of your plans to remove templates.

Reply:
Ah, right. Okay, leave it in, then -- makes the usage closer to what the template-less version will be.